Week3/homework/step2-2.js

```diff
@@ -3,18 +3,29 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+
+  console.log(numbers);
+
+  numbers.map(number => {
```

Comment:
The purpose of the `.map()` method is to create a new array by transforming (i.e. _mapping_) each element of the original through some transformation callback function. Here, you are just iterating through the array. You have no use for the array that `.map()` produces. A simple `.forEach()` will do here. In that case you also can get rid of line 19: there is no need to return anything.

Using a `.map()` method like you do here is confusing to the reader of your code as there will be some expectation that some mapping is required, while there is no such requirement.